lib/Backend/arm64/ARM64Encoder.h

```diff
@@ -3596,7 +3596,7 @@ EmitLdrRegister64(
     Arm64RegisterParam Index
     )
 {
-    return EmitLdrStrRegisterCommon(Emitter, Dest, Addr, Index, 2, 0xf8600800);
+    return EmitLdrStrRegisterCommon(Emitter, Dest, Addr, Index, 3, 0xf8600800);
```

Comment:
>, [](start = 65, length = 1)

Can we replace these 0,1,2,3 etc with a explanatory enum value

Reply:
Yeah. Updated PR